src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -38,10 +40,10 @@ public static IApplicationBuilder UseApplicationInsightsExceptionTelemetry(this
             return app.UseMiddleware<ExceptionTrackingMiddleware>();
         }
 
-        public static void AddApplicationInsightsTelemetry(this IServiceCollection services, IConfiguration config, bool reuseActiveConfig = true)
+        public static void AddApplicationInsightsTelemetry(this IServiceCollection services, IConfiguration config, bool disableDefaultAdaptiveSampling = false)
```

Comment:
Sure, let us discuss about using the config. However, "reuseActiveConfig" was never available for customers. It is always in develop, and we planned to have it in this merge to master.